actions/installer/spec/index.test.ts

```diff
@@ -0,0 +1,56 @@
+const exec = require("@actions/exec");
```

Comment:
nit: we haven't been good about this but add the copyright header with 'SLSA Authors' as the author.

See the header here for an example: https://github.com/slsa-framework/slsa-github-generator/blob/main/slsa/buildtype.go

same for other code files.